ansible/roles/ci/doc/setup/shippable.md

```diff
@@ -0,0 +1,26 @@
+# Shippable Server
+
+## How to setup
+
+The following repository can be used as an example of [Shippable](https://wwww.shippable.com/) configuration [build servers check](https://github.com/shadow-robot/build-servers-check)
```

Comment:
+The following repository can be used as an example of a [Shippable](https://wwww.shippable.com/) configuration: [build servers check](https://github.com/shadow-robot/build-servers-check).